integration-tests/hibernate-search-elasticsearch/src/main/java/io/quarkus/test/hibernate/search/elasticsearch/search/Person.java

```diff
@@ -0,0 +1,71 @@
+package io.quarkus.test.hibernate.search.elasticsearch.search;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+
+import org.hibernate.search.engine.backend.document.model.dsl.ObjectFieldStorage;
+import org.hibernate.search.engine.backend.types.Sortable;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.FullTextField;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.Indexed;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.IndexedEmbedded;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.KeywordField;
+
+@Entity
+@Indexed
+public class Person {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "personSeq")
+    private Long id;
+
+    @FullTextField(analyzer = "standard")
+    @KeywordField(name = "name_sort", normalizer = "lowercase", sortable = Sortable.YES)
+    private String name;
+
+    @ManyToOne(cascade = CascadeType.ALL, fetch = FetchType.LAZY)
+    @IndexedEmbedded(storage = ObjectFieldStorage.NESTED)
```

Comment:
FYI nested storage does not make much sense on single-valued properties. But I suppose the goal was just to test that nested storage doesn't fail.

Reply:
Yes, exactly.